go/dataspike/dataspike.go

```diff
@@ -0,0 +1,273 @@
+package dataspike
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+)
+
+const tokenHeader = "ds-api-token"
+
+type IDataspikeClient interface {
+	GetVerificationByID(string) (*Verification, error)
```

Comment:
```suggestion
	GetVerificationByID(UUID) (*Verification, error)
```
`github.com/gofrs/uuid`